<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogDummyMotor</class>
 <widget class="QDialog" name="dialogDummyMotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>318</width>
    <height>239</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>318</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Properties</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSpeed">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Speed">
      <property name="suffix">
       <string> mm/s</string>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelAcc">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Acceleration</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Accel">
      <property name="suffix">
       <string> mm/sÂ²</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>211</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Axis</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_EnableX">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_EnableA">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_EnableY">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_EnableB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable B</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_EnableZ">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable Z</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_EnableC">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Enable C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonCalib">
      <property name="text">
       <string>Calibrate</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>12</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>42</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBox_Speed</tabstop>
  <tabstop>doubleSpinBox_Accel</tabstop>
  <tabstop>checkBox_EnableX</tabstop>
  <tabstop>checkBox_EnableY</tabstop>
  <tabstop>checkBox_EnableZ</tabstop>
  <tabstop>checkBox_EnableA</tabstop>
  <tabstop>checkBox_EnableB</tabstop>
  <tabstop>checkBox_EnableC</tabstop>
  <tabstop>pushButtonCalib</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogDummyMotor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogDummyMotor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
